service/gradle.lockfile

```diff
@@ -0,0 +1,250 @@
+net.minidev:accessors-smart:2.4.8=testCompileClasspath,testRuntimeClasspath
+net.minidev:json-smart:2.4.8=testCompileClasspath,testRuntimeClasspath
+net.sf.saxon:Saxon-HE:10.5=spotbugs
+org.apache.bcel:bcel:6.5.0=spotbugs
+org.apache.commons:commons-collections4:4.4=productionRuntimeClasspath,runtimeClasspath,testRuntimeClasspath
+org.apache.commons:commons-compress:1.21=productionRuntimeClasspath,runtimeClasspath,testRuntimeClasspath
+org.apache.commons:commons-dbcp2:2.9.0=compileClasspath,productionRuntimeClasspath,runtimeClasspath,testCompileClasspath,testRuntimeClasspath
+org.apache.commons:commons-lang3:3.12.0=compileClasspath,productionRuntimeClasspath,runtimeClasspath,spotbugs,testCompileClasspath,testRuntimeClasspath
+org.apache.commons:commons-pool2:2.11.1=compileClasspath,productionRuntimeClasspath,runtimeClasspath,testCompileClasspath,testRuntimeClasspath
+org.apache.commons:commons-text:1.9=spotbugs
+org.apache.httpcomponents:httpclient:4.5.13=compileClasspath,productionRuntimeClasspath,runtimeClasspath,testCompileClasspath,testRuntimeClasspath
+org.apache.httpcomponents:httpcore:4.4.15=compileClasspath,productionRuntimeClasspath,runtimeClasspath,testCompileClasspath,testRuntimeClasspath
+org.apache.logging.log4j:log4j-api:2.17.2=compileClasspath,productionRuntimeClasspath,runtimeClasspath,testCompileClasspath,testRuntimeClasspath
+org.apache.logging.log4j:log4j-to-slf4j:2.17.2=compileClasspath,productionRuntimeClasspath,runtimeClasspath,testCompileClasspath,testRuntimeClasspath
+org.apache.oltu.oauth2:org.apache.oltu.oauth2.client:1.0.1=compileClasspath,productionRuntimeClasspath,runtimeClasspath,testCompileClasspath,testRuntimeClasspath
+org.apache.oltu.oauth2:org.apache.oltu.oauth2.common:1.0.1=compileClasspath,productionRuntimeClasspath,runtimeClasspath,testCompileClasspath,testRuntimeClasspath
+org.apache.tomcat.embed:tomcat-embed-core:9.0.65=compileClasspath,productionRuntimeClasspath,runtimeClasspath,testCompileClasspath,testRuntimeClasspath
+org.apache.tomcat.embed:tomcat-embed-el:9.0.65=compileClasspath,productionRuntimeClasspath,runtimeClasspath,testCompileClasspath,testRuntimeClasspath
+org.apache.tomcat.embed:tomcat-embed-websocket:9.0.65=compileClasspath,productionRuntimeClasspath,runtimeClasspath,testCompileClasspath,testRuntimeClasspath
+org.apiguardian:apiguardian-api:1.1.2=testCompileClasspath
+org.aspectj:aspectjweaver:1.9.7=compileClasspath,productionRuntimeClasspath,runtimeClasspath,testCompileClasspath,testRuntimeClasspath
+org.assertj:assertj-core:3.22.0=testCompileClasspath,testRuntimeClasspath
+org.attoparser:attoparser:2.0.5.RELEASE=compileClasspath,productionRuntimeClasspath,runtimeClasspath,testCompileClasspath,testRuntimeClasspath
+org.bitbucket.b_c:jose4j:0.7.12=productionRuntimeClasspath,runtimeClasspath,testRuntimeClasspath
+org.bouncycastle:bcpkix-jdk15on:1.67=productionRuntimeClasspath,runtimeClasspath,testRuntimeClasspath
```

Comment:
@iliadsh minor comment on these dependencies

Please note that multiple versions of the same or similar dependencies are being added to the service. Example- 

jdk18 and 15 versions added -> is this required and as intended? We are configuring the project to use java 17
```
org.bouncycastle:bcprov-jdk15on:1.67=productionRuntimeClasspath,runtimeClasspath,testRuntimeClasspath
org.bouncycastle:bcprov-jdk18on:1.71=productionRuntimeClasspath,runtimeClasspath,testRuntimeClasspath
```

scala 12 and 13 versions are included. Generally, we exclude the _12 versions explicitly across terra repos.  
```
org.broadinstitute.dsde.workbench:sam-client_2.12:0.1-61135c7=compileClasspath,productionRuntimeClasspath,runtimeClasspath,testCompileClasspath,testRuntimeClasspath
org.broadinstitute.dsde.workbench:sam-client_2.13:0.1-33f3550=productionRuntimeClasspath,runtimeClasspath,testRuntimeClasspath
```

similar at 
```
org.checkerframework:checker-qual:3.22.2=productionRuntimeClasspath,runtimeClasspath,testRuntimeClasspath
org.checkerframework:checker-qual:3.5.0=liquibaseRuntime
```

```
org.jetbrains.kotlin:kotlin-stdlib-jdk7:1.6.21=compileClasspath,productionRuntimeClasspath,runtimeClasspath,testCompileClasspath,testRuntimeClasspath
org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.6.21=compileClasspath,productionRuntimeClasspath,runtimeClasspath,testCompileClasspath,testRuntimeClasspath
org.jetbrains.kotlin:kotlin-stdlib:1.6.21=compileClasspath,productionRuntimeClasspath,runtimeClasspath,testCompileClasspath,testRuntimeClasspath
```

etc. more further down 

Its best to clean up the dependencies and avoid conflicting versions including in the same service. This may need parsing through few console logs to identify and exclude the ones we dont need / want. 

If you need help debugging this, feel free to set up a meeting, happy to help

Reply:
@dexamundsen Purged as many direct duplicates as I could without breaking things. How does it look?